tests/system/python/api/test_audit.py

```diff
@@ -110,5 +148,26 @@ def test_create_audit_entry(self, fledge_url, payload, total_count):
         assert 200 == r.status
         r = r.read().decode()
         jdoc = json.loads(r)
+
+        # Storage category when with SQLite plugin, creates a child category `sqlite` hence one additional CONAD entry.
+        # CONAD, INFORMATION, '{{"name":"sqlite", ...}'
+        # so total_count is being decresed in case of 'postgres' storage engine
+        if storage_plugin == 'postgres':
+           total_count = total_count -1
+
         assert len(jdoc), "No data found"
         assert total_count == jdoc['totalCount']
+
+    @pytest.mark.parametrize("payload", [
+        ({"source": "LOGGN_X", "severity": "warning", "details": {"message": "Engine oil pressure low"}}),
+        ({"source": "LOG_X", "severity": "information", "details": {"message": "Engine oil pressure is okay."}})
+    ])
+    def test_create_nonexistent_log_code_audit_entry(self, fledge_url, payload, storage_plugin):
```

Comment:
I see storage_plugin parser option is not used for the test. Does it require?

Reply:
No, I had a skip with storage_plugin sqlite (TODO: merge foreign key support first) then removed that to let it fail but forgot to remove. I will remove it.